package.json

```diff
@@ -18,25 +18,27 @@
   ],
   "scripts": {
     "clean": "rm -rf dist && mkdir dist",
-    "lint": "./node_modules/.bin/eslint src/** tests/**",
```

Comment:
`./node_modules/.bin` prefix is unneeded.
Oddly, `tests` was incorrect - the path was `test`. This broke linting... (maybe latest eslint is more strict)